pulsar-client-cpp/include/pulsar/Consumer.h

```diff
@@ -390,6 +390,11 @@ class PULSAR_PUBLIC Consumer {
      */
     bool isConnected() const;
 
+    /**
+     * Get the latest message id for the consumer's topic.
+     */
+    virtual void getLastMessageIdAsync(BrokerGetLastMessageIdCallback callback);
```

Comment:
`BrokerGetLastMessageIdCallback` is an internal callback definition defined in `ConsumerImpl.h`, you should define your own callback. See `include/pulsar/Reader.h` for how it defines the `getLastMessageIdAsync`. And there is no need to make it `virtual`.

Reply:
Thank you for your guidance! I'll take a look at that.